Remove mandatory Google AdMob dependency and configuration requirement for Xamarin Android library

Version 6.0.4 of the Xamarin Android nuget package requires a Google AdMob app ID configured and a run time exception is thrown if it is not configured. This is not required when using the native Android Localytics library and we'd like to be able to exclude this when using the Xamarin library.

To reproduce run the latest sample android app.

The run time error that is thrown is shown below:

```
[MonoDroid] UNHANDLED EXCEPTION:
[MonoDroid] Java.Lang.RuntimeException: Unable to get provider com.google.android.gms.ads.MobileAdsInitProvider: java.lang.IllegalStateException: 
[MonoDroid] 
[MonoDroid] ******************************************************************************
[MonoDroid] * The Google Mobile Ads SDK was initialized incorrectly. AdMob publishers    *
[MonoDroid] * should follow the instructions here:                                       *
[MonoDroid] * https://googlemobileadssdk.page.link/admob-android-update-manifest         *
[MonoDroid] * to add a valid App ID inside the AndroidManifest.                          *
[MonoDroid] * Google Ad Manager publishers should follow instructions here:              *
[MonoDroid] * https://googlemobileadssdk.page.link/ad-manager-android-update-manifest.   *
[MonoDroid] ******************************************************************************
[MonoDroid] 
[MonoDroid]  ---> Java.Lang.IllegalStateException: 
[MonoDroid] 
[MonoDroid] ******************************************************************************
[MonoDroid] * The Google Mobile Ads SDK was initialized incorrectly. AdMob publishers    *
[MonoDroid] * should follow the instructions here:                                       *
[MonoDroid] * https://googlemobileadssdk.page.link/admob-android-update-manifest         *
[MonoDroid] * to add a valid App ID inside the AndroidManifest.                          *
[MonoDroid] * Google Ad Manager publishers should follow instructions here:              *
[MonoDroid] * https://googlemobileadssdk.page.link/ad-manager-android-update-manifest.   *
[MonoDroid] ******************************************************************************
[MonoDroid] 
[MonoDroid] 
[MonoDroid]    --- End of inner exception stack trace ---
[MonoDroid]   --- End of managed Java.Lang.RuntimeException stack trace ---
[MonoDroid] java.lang.RuntimeException: Unable to get provider com.google.android.gms.ads.MobileAdsInitProvider: java.lang.IllegalStateException: 
[MonoDroid] 
[MonoDroid] ******************************************************************************
[MonoDroid] * The Google Mobile Ads SDK was initialized incorrectly. AdMob publishers    *
[MonoDroid] * should follow the instructions here:                                       *
[MonoDroid] * https://googlemobileadssdk.page.link/admob-android-update-manifest         *
[MonoDroid] * to add a valid App ID inside the AndroidManifest.                          *
[MonoDroid] * Google Ad Manager publishers should follow instructions here:              *
[MonoDroid] * https://googlemobileadssdk.page.link/ad-manager-android-update-manifest.   *
[MonoDroid] ******************************************************************************
[MonoDroid] 
[MonoDroid] 
[MonoDroid] 	at android.app.ActivityThread.installProvider(ActivityThread.java:7244)
[MonoDroid] 	at android.app.ActivityThread.installContentProviders(ActivityThread.java:6780)
[MonoDroid] 	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6697)
[MonoDroid] 	at android.app.ActivityThread.access$1300(ActivityThread.java:237)
[MonoDroid] 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1913)
[MonoDroid] 	at android.os.Handler.dispatchMessage(Handler.java:106)
[MonoDroid] 	at android.os.Looper.loop(Looper.java:223)
[MonoDroid] 	at android.app.ActivityThread.main(ActivityThread.java:7656)
[MonoDroid] 	at java.lang.reflect.Method.invoke(Native Method)
[MonoDroid] 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
[MonoDroid] 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
[MonoDroid] Caused by: java.lang.IllegalStateException: 
[MonoDroid] 
[MonoDroid] ******************************************************************************
[MonoDroid] * The Google Mobile Ads SDK was initialized incorrectly. AdMob publishers    *
[MonoDroid] * should follow the instructions here:                                       *
[MonoDroid] * https://googlemobileadssdk.page.link/admob-android-update-manifest         *
[MonoDroid] * to add a valid App ID inside the AndroidManifest.                          *
[MonoDroid] * Google Ad Manager publishers should follow instructions here:              *
[MonoDroid] * https://googlemobileadssdk.page.link/ad-manager-android-update-manifest.   *
[MonoDroid] ******************************************************************************
[MonoDroid] 
[MonoDroid] 
[MonoDroid] 	at com.google.android.gms.internal.ads.zzzt.attachInfo(com.google.android.gms:play-services-ads-lite@@19.7.0:31)
[MonoDroid] 	at com.google.android.gms.ads.MobileAdsInitProvider.attachInfo(com.google.android.gms:play-services-ads-lite@@19.7.0:3)
[MonoDroid] 	at android.app.ActivityThread.installProvider(ActivityThread.java:7239)
[MonoDroid] 	... 10 more
[MonoDroid] 
```

--- a/LocalyticsXamarin/LocalyticsXamarin.Android/LocalyticsXamarin.Android.csproj
+++ b/LocalyticsXamarin/LocalyticsXamarin.Android/LocalyticsXamarin.Android.csproj
@@ -83,17 +83,6 @@
       <HintPath>..\packages\Xamarin.GooglePlayServices.Location.60.1142.1\lib\MonoAndroid80\Xamarin.GooglePlayServices.Location.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="Xamarin.GooglePlayServices.Ads.Lite">
-      <HintPath>..\packages\Xamarin.GooglePlayServices.Ads.Lite.60.1142.1\lib\MonoAndroid80\Xamarin.GooglePlayServices.Ads.Lite.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
-    <Reference Include="Xamarin.GooglePlayServices.Gass">
-      <HintPath>..\packages\Xamarin.GooglePlayServices.Gass.60.1142.1\lib\MonoAndroid80\Xamarin.GooglePlayServices.Gass.dll</HintPath>
-    </Reference>
-    <Reference Include="Xamarin.GooglePlayServices.Ads">
-      <HintPath>..\packages\Xamarin.GooglePlayServices.Ads.60.1142.1\lib\MonoAndroid80\Xamarin.GooglePlayServices.Ads.dll</HintPath>
-      <Private>False</Private>
-    </Reference>
     <Reference Include="Xamarin.AndroidX.Activity, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\Xamarin.AndroidX.Activity.1.2.0.1\lib\monoandroid90\Xamarin.AndroidX.Activity.dll</HintPath>
     </Reference>
@@ -253,7 +242,6 @@
   <Import Project="..\packages\Xamarin.Firebase.Messaging.60.1142.1\build\MonoAndroid80\Xamarin.Firebase.Messaging.targets" Condition="Exists('..\packages\Xamarin.Firebase.Messaging.60.1142.1\build\MonoAndroid80\Xamarin.Firebase.Messaging.targets')" />
   <Import Project="..\packages\Xamarin.Build.Download.0.4.11\build\Xamarin.Build.Download.targets" Condition="Exists('..\packages\Xamarin.Build.Download.0.4.11\build\Xamarin.Build.Download.targets')" />
   <Import Project="..\packages\Xamarin.GooglePlayServices.Base.60.1142.1\build\MonoAndroid80\Xamarin.GooglePlayServices.Base.targets" Condition="Exists('..\packages\Xamarin.GooglePlayServices.Base.60.1142.1\build\MonoAndroid80\Xamarin.GooglePlayServices.Base.targets')" />
-  <Import Project="..\packages\Xamarin.GooglePlayServices.Gass.60.1142.1\build\MonoAndroid80\Xamarin.GooglePlayServices.Gass.targets" Condition="Exists('..\packages\Xamarin.GooglePlayServices.Gass.60.1142.1\build\MonoAndroid80\Xamarin.GooglePlayServices.Gass.targets')" />
   <Import Project="..\packages\Xamarin.AndroidX.Activity.1.2.0.1\build\monoandroid9.0\Xamarin.AndroidX.Activity.targets" Condition="Exists('..\packages\Xamarin.AndroidX.Activity.1.2.0.1\build\monoandroid9.0\Xamarin.AndroidX.Activity.targets')" />
   <Import Project="..\packages\Xamarin.AndroidX.Annotation.1.1.0.9\build\monoandroid9.0\Xamarin.AndroidX.Annotation.targets" Condition="Exists('..\packages\Xamarin.AndroidX.Annotation.1.1.0.9\build\monoandroid9.0\Xamarin.AndroidX.Annotation.targets')" />
   <Import Project="..\packages\Xamarin.AndroidX.Annotation.Experimental.1.0.0.9\build\monoandroid9.0\Xamarin.AndroidX.Annotation.Experimental.targets" Condition="Exists('..\packages\Xamarin.AndroidX.Annotation.Experimental.1.0.0.9\build\monoandroid9.0\Xamarin.AndroidX.Annotation.Experimental.targets')" />

--- a/LocalyticsXamarin/LocalyticsXamarin.Android/packages.config
+++ b/LocalyticsXamarin/LocalyticsXamarin.Android/packages.config
@@ -60,13 +60,9 @@
   <package id="Xamarin.Google.Android.DataTransport.TransportRuntime" version="2.2.5" targetFramework="monoandroid11.0" requireReinstallation="true" />
   <package id="Xamarin.Google.Dagger" version="2.27.0" targetFramework="monoandroid11.0" requireReinstallation="true" />
   <package id="Xamarin.Google.Guava.ListenableFuture" version="1.0.0.2" targetFramework="monoandroid11.0" requireReinstallation="true" />
-  <package id="Xamarin.GooglePlayServices.Ads.Base" version="119.7.0" targetFramework="monoandroid11.0" requireReinstallation="true" />
-  <package id="Xamarin.GooglePlayServices.Ads.Identifier" version="117.0.0" targetFramework="monoandroid11.0" requireReinstallation="true" />
-  <package id="Xamarin.GooglePlayServices.Ads.Lite" version="119.7.0" targetFramework="monoandroid11.0" requireReinstallation="true" />
   <package id="Xamarin.GooglePlayServices.Base" version="117.6.0" targetFramework="monoandroid11.0" requireReinstallation="true" />
   <package id="Xamarin.GooglePlayServices.Basement" version="117.6.0" targetFramework="monoandroid11.0" requireReinstallation="true" />
   <package id="Xamarin.GooglePlayServices.CloudMessaging" version="116.0.0" targetFramework="monoandroid11.0" requireReinstallation="true" />
-  <package id="Xamarin.GooglePlayServices.Gass" version="119.7.0" targetFramework="monoandroid11.0" requireReinstallation="true" />
   <package id="Xamarin.GooglePlayServices.Measurement.Base" version="118.0.2" targetFramework="monoandroid11.0" requireReinstallation="true" />
   <package id="Xamarin.GooglePlayServices.Measurement.Sdk.Api" version="118.0.2" targetFramework="monoandroid11.0" requireReinstallation="true" />
   <package id="Xamarin.GooglePlayServices.Stats" version="117.0.0" targetFramework="monoandroid11.0" requireReinstallation="true" />

